bfly/CoreLayer/AccessLayer/QueryLayer/UtilityLayer/Structures.py

```diff
@@ -4,19 +4,19 @@
 class NamelessStruct(object):
     VALUE = None
     def __init__(self,**_keywords):
-        # Begin making the list
-        all_list = _keywords.get('LIST',[])
+        # Begin a new list of all keyword NAMES
+        all_list = _keywords.get('LIST',[])[:]
         # Add other keywords to list
         for key in _keywords:
             keyval = _keywords[key]
             setattr(self,key,keyval)
             # Put all lists into list
```

Comment:
The below changes are just to improve code legibility. They have no functional effect.